other/materials_designer/create_interface_with_min_strain_zsl.ipynb

```diff
@@ -50,8 +50,27 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    FILM_INDEX = 1
</pre><p> </p><p><code>FILM_INDEX = 1  # Index in the list of materials, to access as materials[FILM_INDEX]</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>ok</p>